quiche/src/lib.rs

```diff
@@ -4861,6 +4861,12 @@ impl Connection {
         self.dgram_send_queue.byte_size()
     }
 
+    /// Returns whether or not the DATAGRAM send queue is full.
+    #[inline]
+    pub fn dgram_send_queue_full(&self) -> bool {
```

Comment:
maybe prefix with `is_` to be more consistent with others ?
```suggestion
    pub fn is_dgram_send_queue_full(&self) -> bool {
```